frontends/web/src/contexts/DarkmodeProvider.tsx

```diff
@@ -43,16 +45,22 @@ export const DarkModeProvider = ({ children }: TProps) => {
     getConfig()
       .then(config => {
         // use config if it exists
-        if ('darkmode' in config.frontend) {
+        if (!!config.frontend && 'darkmode' in config.frontend) {
```

Comment:
This missing check was causing a weird behavior due to js runtime exceptions on the first fresh start of the app, with no config file.